tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd

```diff
@@ -344,9 +368,6 @@ help documentation for this function by running the code: `?group_by()`
 
 ```{r compare-classes}
 
-# View help file for group_by() function
-?group_by()
```

Comment:
Why remove this?

Reply:
It opened up a browser tab every time I knit it! Made a separate code chunk with eval=FALSE

Reply:
Ahhh yes that makes sense.